### ISSUE TITLE
Setting media_library.config.max_file_size has no effect

### ISSUE THREAD

**Issue by repomaa** [StaticJsCMS/static-cms#486]
I'm using local file uploads so no external media library. I set the `max_file_size` option in my config but it has no effect. I set it to `10000000` but it still says "File too large" telling me the limit is 512 kB.

**Comment by ky-is** [comment]
`max_file_size` is working for me in my `config.yml` like this, maybe it can help you:
```yaml
collections:
  - label: "Image"
    fields:
      - { label: "Image", name: "image", widget: "image", media_library: { config: { max_file_size: 0 } } }
  # ...
```
(0 works as unlimited)

imo the issue here is the default is too low at 512000 (https://github.com/StaticJsCMS/static-cms/blob/main/packages/core/src/components/MediaLibrary/MediaLibrary.tsx#L192).

Netlify says [in their docs](https://docs.netlify.com/large-media/requirements-and-limitations/) it's intended for files up to 100mb. Although that will vary by provider, something like that makes more sense as a default to me, if not unlimited.

**Comment by shawntax** [comment]
Same issue here.

I found the [relevant part of the docs](https://staticjscms.netlify.app/docs/beta-features#image-widget-file-size-limit) that points out @ky-is 's solution above. It's odd that you can only set a max file size only in relation to an `image` widget, and not for the local media library generally, given that you can access the media library and upload files independently of creating a new collection item.

**Comment by KaneFreeman** [comment]
This will be resolved in `v2.0.0`.
